api/v1alpha1/contentlibraryprovider_types.go

```diff
@@ -17,6 +17,9 @@ type ContentLibraryProviderSpec struct {
 // ContentLibraryProviderStatus defines the observed state of ContentLibraryProvider
 // Can include fields indicating when was the last time VM images were updated from a library
 type ContentLibraryProviderStatus struct {
+	// DisplayName describes the display name of the ContentLibraryProvider, which is the name of the vSphere content library.
+	// +optional
+	DisplayName string `json:"displayName,omitempty"`
```

Comment:
Let's do the DisplayName of the ContentLIbrary as a followup story, so this can be removed